torchx/c_src/torchx.cpp

```diff
@@ -786,6 +786,32 @@ NIF(solve)
   TENSOR(torch::linalg_solve(*tensorA, *tensorB));
 }
 
+NIF(conv)
+{
+  TENSOR_PARAM(0, tensorA);
+  TENSOR_PARAM(1, tensorB);
+
+  LIST_PARAM(2, std::vector<int64_t>, stride);
+  LIST_PARAM(3, std::vector<int64_t>, padding);
+  LIST_PARAM(4, std::vector<int64_t>, dilation);
+  PARAM(5, bool, transposed);
+  PARAM(6, int64_t, groups);
+
+  c10::optional<at::Tensor> bias_tensor;
+
+  std::vector<int64_t> output_padding;
+  // aten::_convolution(Tensor input, Tensor weight, Tensor? bias,
+  //      int[] stride, int[] padding, int[] dilation, bool transposed,
+  //      int[] output_padding, int groups, bool benchmark, bool deterministic, bool cudnn_enabled, bool allow_tf32) -> Tensor
+  TENSOR(at::_convolution(*tensorA, *tensorB, bias_tensor,
+    stride, padding, dilation, transposed, output_padding, groups,
+    true,   // benchmark
+    true,   // deterministic
```

Comment:
Just a note that for performance this should probably be set to false, but we should later consider or see how we can add a deterministic API to Nx compilers/backends to guarantee consistent results in different algorithms

Reply:
ok